gammapy/makers/utils.py

```diff
@@ -101,6 +104,40 @@ def _get_fov_coords(pointing, irf, geom, use_region_center=True, obstime=None):
     return coords
 
 
+def compute_rotation_time_step(rotation, pointing_altaz):
+    """
+    Compute the time such that the FoV associated to a fixed RaDec position rotates by 'rotation' in AltAz frame.
+    It assumes that the rotation rate at the provided pointing is a good estimate of the rotation rate over the full
+    output duration (first order approximation).
+
+
+    Parameters
+    ----------
+    rotation : `~astropy.units.Quantity`
+        Rotation angle.
+    pointing_altaz : `~astropy.coordinates.SkyCoord`
+        Pointing direction.
+
+    Returns
+    -------
+    duration : `~astropy.units.Quantity`
+        Time associated with the requested rotation.
+    """
+    denom = (
+        EARTH_ANGULAR_VELOCITY
+        * np.cos(pointing_altaz.location.lat.rad)
+        * np.abs(np.cos(pointing_altaz.az.rad))
+    )
+    if denom.value == 0:
+        if np.cos(pointing_altaz.location.lat.rad) == 0:
+            # Assuming that the observatory location is fixed during one observation, the time step can be infinite
+            return np.inf * u.s
```

Comment:
@registerrier: should we add an info message for these 2 cases?
@gabemery: it looks that these 2 cases are not tested. Is it needed?

Reply:
They are not tested because numpy.cos is never exactly zero. So in application it is currently useless.
And impossible to test. But I kept it in case numpy behaviour changes in the future or is architecture dependent. 

I am open to removing it, since I don't like having untested lines.